lib/samples/apply_simple_renderer_to_feature_layer/apply_simple_renderer_to_feature_layer_sample.dart

```diff
@@ -27,69 +27,93 @@ class ApplySimpleRendererToFeatureLayerSample extends StatefulWidget {
 
 class _ApplySimpleRendererToFeatureLayerSampleState
     extends State<ApplySimpleRendererToFeatureLayerSample> {
-  final _mapViewController = ArcGISMapView.createController();
   late FeatureLayer _featureLayer;
-  final _map = ArcGISMap.withBasemapStyle(BasemapStyle.arcGISTopographic);
-
-  @override
-  void initState() {
-    super.initState();
-
-    final uri = Uri.parse(
-        'https://services.arcgis.com/V6ZHFr6zdgNZuVG0/arcgis/rest/services/Landscape_Trees/FeatureServer/0');
-    final serviceFeatureTable = ServiceFeatureTable.withUri(uri);
-    _featureLayer = FeatureLayer.withFeatureTable(serviceFeatureTable);
-    _map.operationalLayers.add(_featureLayer);
-    _map.initialViewpoint = Viewpoint.fromCenter(
-      ArcGISPoint(
-        x: -9177250,
-        y: 4247200,
-        spatialReference: SpatialReference.webMercator,
-      ),
-      scale: 9500,
-    );
-    _mapViewController.arcGISMap = _map;
-  }
+  final _mapViewController = ArcGISMapView.createController();
```

Comment:
it's customary these days to put a comment before this line, like `  // create a map view controller`

Reply:
All properties have been commented.